exes/ec/inc/public/tfc/ec/devices/util.hpp

```diff
@@ -50,8 +50,9 @@ template <ecx::index_t idx,
           auto... value_t_construct_params>
 struct glz::meta<tfc::ec::util::setting<idx, name_value, desc, value_t, value_t_construct_params...>> {
   using setting = tfc::ec::util::setting<idx, name_value, desc, value_t, value_t_construct_params...>;
+  static constexpr std::string_view description{ desc };
   static constexpr auto value{
-    glz::object("value", &setting::value, "index", &setting::index, "name", &setting::name_v, "desc", &setting::desc_v)
+    glz::object("value", &setting::value, description)  // todo concat description, index and name
```

Comment:
Better,
This might need changing depending on the UI form library used.

Reply:
will be addressed in future PR, reverted the changes